sdk/personalizer/azure-ai-personalizer/tests.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      BuildTargetingString: azure-ai-personalizer
+      ServiceDirectory: personalizer
+      MatrixReplace:
+        - TestSamples=.*/true
+      Clouds: Canary
```

Comment:
I don't think we need to run tests in Canary (it doesn't look like we do in the other languages).

```suggestion
```